ansible/roles/provision-vm/tasks/fedora-coreos.yml

```diff
@@ -40,4 +40,4 @@
   wait_for_connection:
     delay: 10
     sleep: 10
-    timeout: 60
+    timeout: 120
```

Comment:
Why do we need to change the timeout for fedora? Is it taking longer to process the playbooks?

Reply:
I'm currently trying to figure that out - the error seems to be that it hasn't fully rebooted yet at this point. Not sure what the root cause is currently